### DISCUSSION TITLE
dataset

### DISCUSSION THREAD

**Discussion by andyye1999** [jhauret/eben#5] [Q&A] [answered]
Hello author, after reading your article, I thought of my current application scenario which involves mapping bone conduction speech signals to air-conducted speech signals. However, the dataset is currently limited. How does the size of the dataset affect the performance of an EBNE network?

**Comment by jhauret** [comment, marked as answer]
Hi,

Our approach can be applied to any body-conduction microphone including bone, in-ear, and throat microphones.

If your dataset is limited, you can try to perform a pre-training on simulated data (just as we did by applying a low-pass filter on clean speech with roughly the same characteristics as the mic you're tackling).

The size of the finetuning dataset is a question that we are currently exploring. We are planning to record 50 hours of air and body-conducted speech. We believe it will be sufficient with well-adapted hyperparameters.

Good luck with your application!

**Reply by andyye1999** [reply]
I have a dataset of about 5 hours, do you think the model will converge?

**Reply by jhauret** [reply]
Idk, deep learning is an experimental science, so you should give it a try. With adequate pre-training, it might work. 😉

**Reply by andyye1999** [reply]
i am working on it  and hope it can work 
I am very interested in your research and would like to express my gratitude.

**Reply by andyye1999** [reply]
> Hi,
> 
> Our approach can be applied to any body-conduction microphone including bone, in-ear, and throat microphones.
> 
> If your dataset is limited, you can try to perform a pre-training on simulated data (just as we did by applying a low-pass filter on clean speech with roughly the same characteristics as the mic you're tackling).
> 
> The size of the finetuning dataset is a question that we are currently exploring. We are planning to record 50 hours of air and body-conducted speech. We believe it will be sufficient with well-adapted hyperparameters.
> 
> Good luck with your application!

I previously used a mapping method similar to the speech enhancement model DCCRN and found that it could only map up to 4000Hz. It was unable to map frequencies between 4000Hz and 8000Hz. Currently, I am trying your model.

**Reply by andyye1999** [reply]
> The size of the finetuning dataset is a question that we are currently exploring. We are planning to record 50 hours of air and body-conducted speech. We believe it will be sufficient with well-adapted hyperparameters.


I have found some datasets [data1](https://github.com/elevoctech/ESMB-corpus) But in the country where I am located, it is not possible to download such a large dataset :(  You can download this dataset and take a look at its data quality. Interestingly, we have previously discovered the same dataset. [data2](https://github.com/wangmou21/abcs/issues/1)
